### PR TITLE
Implemented refresh token and scheduler

### PR DESCRIPTION
**Pull request by hopingsteam**

### CODE REVIEW

File: configs/strava_config.json [hunk 1]
```diff
@@ -1,5 +1,8 @@
 {
     "activities_limit": 162,
     "strava_access_token": "<INSERT_ACCESS_TOKEN>",
-    "strava_club_id": "<INSERT_CLUB_ID>"
+    "strava_client_id": "<INSERT_CLIENT_ID>",
+    "strava_client_secret": "<INSERT_CLIENT_SECRET>",
+    "strava_club_id": "<INSERT_CLUB_ID>",
+    "strava_refresh_token": "<INSERT_REFRESH_TOKEN>"
```
Comment (CiprianAmz) [inline configs/strava_config.json:7]: I think one of those is the `strava_access_token` (which makes it duplicated)

File: strava_leaderboard_extractor/strava_config.py [hunk 2]
```diff
@@ -4,17 +4,23 @@
...
 
 
 def load_strava_config_from_json(file_path: str) -> StravaConfig:
     with open(file_path) as f:
         config_json = dict(json.load(f))
         return StravaConfig(
             access_token=config_json["strava_access_token"],
-            club_id=config_json["strava_club_id"],
             activities_limit=config_json.get("activities_limit", None),
+            client_id=int(config_json["strava_client_id"]),
```
Comment (CiprianAmz) [inline strava_leaderboard_extractor/strava_config.py:22]: It seems that we are not using `client_id`. Better remove it

**Review by CiprianAmz — APPROVED** [review]

File: strava_leaderboard_extractor/strava_config.py (hunk 2 above)
Comment (hopingsteam) [reply strava_leaderboard_extractor/strava_config.py:22]: We are using it in `tomita_strava.py:118` in the function `refresh_access_token()`

File: configs/strava_config.json (hunk 1 above)
Comment (hopingsteam) [reply configs/strava_config.json:7]: Nothing is duplicated

**Pull request merged**